chat-widget/webpack.config.cjs

```diff
@@ -22,6 +24,7 @@ module.exports = {
         type: "filesystem",
     },
     devtool: "eval-cheap-source-map", // this is the fastest source map option for development
+    // devtool: false, // to disable source maps for production build
```

Comment:
amm... this doesnt go to production, is just for the sample app for testing we may want to keep it